quicktests/overlaying/playground.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <script type = "text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+        <script type = "text/javascript" src="http://d3js.org/d3.v3.min.js" charset="utf-8"></script>
+        <script type = "text/javascript" src="https://rawgithub.com/palantir/plottable/develop/plottable.js"></script>
+        <script type = "text/javascript" src="http://rawgithub.com/palantir/plottable/develop/quicktests/exampleUtil.js"></script>
+        <script type = "text/javascript" src="http://localhost:9999/quicktests/overlaying/playground.js"></script>
+        <link rel="stylesheet" type="text/css" href="https://rawgithub.com/palantir/plottable/master/plottable.css">
+
+
+    </head>
+    <body>
+        <div align="center">
+        <br><br>
+        <svg id="playground" width="400" height="400"></svg>
+        </div>
+
+        <br>
+
+        <div align="center">
+            <span> width (px): </span>
+            <input type="text" id="svg_w" rows="1" cols="10" value="400"></textarea>
+            <span> height (px): </span>
+            <input type="text" id="svg_h" rows="1" cols="10" value="400"></textarea>
+            <button id="changeSizeButton" onclick="svg_size()"> Change Size </button>
```

Comment:
Probably want to change this to "Update" or something because it's the only way to trigger update behavior with the graph. Would also be cool to add a changeListener or something so the button auto-changes to say "update" when you change parameters for the graph.